bin/webpack-configs.js

```diff
@@ -584,7 +590,7 @@ const getExtensionsConfig = ( options = {} ) => {
 			],
 		},
 		plugins: [
-			...sharedPlugins,
+			...getSharedPlugins( { bundleAnalyzerReportTitle: 'Extensions' } ),
```

Comment:
I think we should use Experimental Extensions here. We have Payment Method Extensions above, so we should be more explicit.

Reply:
Thanks @dinhtungdu , you're right, I totally agree. I'll adjust the report titles:
- Extensions -> Experimental Extensions
- Payments -> Payment Method Extensions
so it aligns with Progress Plugin and are distinguishable!